features/smoke-tests/supplier/opportunities.feature

```diff
@@ -34,46 +33,114 @@ Scenario Outline: User can filter by individual status
   Given I am on the /digital-outcomes-and-specialists/opportunities page
   When I note the result_count
   And I check '<status>' checkbox
-  And I click the 'Filter' button
+  And I wait for the page to reload
   Then I see that the stated number of results does not exceed that result_count
   And I see all the opportunities on the page are of the '<status>' status
 
   Examples:
-    | status |
-    | Open   |
-    | Closed |
+    | status       |
+    | Open         |
+    | Closed       |
+    | Awarded      |
+    | Unsuccessful |
+    | Cancelled    |
 
-Scenario Outline: User can filter by both status and lot
+Scenario Outline: User can filter by individual location
   Given I am on the /digital-outcomes-and-specialists/opportunities page
   When I note the result_count
-  And I check '<lot>' checkbox
-  And I check '<status>' checkbox
-  And I click the 'Filter' button
+  And I check '<location>' checkbox
+  And I wait for the page to reload
+  Then I see that the stated number of results does not exceed that result_count
+  And I see all the opportunities on the page are in the '<location>' location
+
+  Examples:
+    | location                       |
+    | Scotland                       |
+    | International (outside the UK) |
+    | Off-site                       |
+
+Scenario Outline: User gets no results for impossible combinations of location and lot
+  Given I am on the /digital-outcomes-and-specialists/opportunities page
+  And I click '<lot>'
+  And I check '<location>' checkbox
+  And I wait for the page to reload
+  Then I see no results
+  # now we backtrack a bit to check the lot "link" has been made unclickable when showing it has no results
+  When I click 'Clear filters'
+  And I wait for the page to reload
+  And I click 'All categories'
+  And I check '<location>' checkbox
```

Comment:
Other times after clicking on a filter, we've need a "wait for the page to reload" step. What's the cases where we do/don't need this?

Reply:
In this particular case we probably should - I think  we might have just been getting lucky so far...

Generally I omit it when it's a link we're clicking.